magefile.go

```diff
@@ -120,9 +120,10 @@ func dryRunPackageRegistry() error {
 	}
 	defer os.Chdir(currentDir)
 
-	err = os.Chdir(buildDir)
+	// Change to the testing directory to run package-registry from there as it contains the config for it.
+	err = os.Chdir("testing")
 	if err != nil {
-		return errors.Wrapf(err, "can't change directory to %s", buildDir)
+		return errors.Wrapf(err, "can't change directory to %s", "testing")
```

Comment:
nit: you can unify this. The `testing` const is mentioned in two places now.

Reply:
agree, done.